shared/polyfills/window.js

```diff
@@ -52,6 +52,14 @@ if (!globalThis.HTMLElement) {
   globalThis.HTMLElement = HTMLElement;
 }
 
+if (!globalThis.HTMLVideoElement) {
+  class HTMLVideoElement extends EventTarget {}
```

Comment:
blocking: I think we're going to need to do some thorough testing to make sure we aren't breaking SSR polyfills here. If we are, this might mean we need to bump up prioritizing how we're handling polyfills on the Open Elements side of the fence.

Reply:
we're already polyfilling related things like HTMLElement and EventTarget above, I'm not sure that this addition will cause any issues.
Currently, the polyfills are run before the rest of mux-video and mux-player.

Reply:
cool then marking as resolved for now. We've already run into some issues, but now that we're discussing those may have all been `document`-rlated.